examples/advanced/federated-statistics/df_stats/README.md

```diff
@@ -102,7 +103,7 @@ nvflare poc --start -p admin
 
 Inside the console, submit the job:
 ```
-submit_job federated_statistics/df_stats/df_stats_job
+submit_job advanced/federated-statistics/df_stats/jobs/df_stats
```

Comment:
^ this path depends on where you expect user to run this command.

Do you have the steps that copy NVFlare/examples/* into the transfer folder?